add templates, CoC and contributing guide (#126)
OpenAccess-AI-Collective/axolotl: README.md

## Contributing 🤝

Please read the [contributing guide](./.github/CONTRIBUTING.md)

> Bugs? Please check the [open issues](https://github.com/OpenAccess-AI-Collective/axolotl/issues/bug) else create a new Issue.

Reviewer: ```suggestion
Bugs? Please check for open issues first before creating a new [Issue](https://github.com/OpenAccess-AI-Collective/axolotl/issues/new).
```

Author: this link will show the open bugs, so my suggested edit might make sense after all?

Reviewer: Yes, I think yours is better.
